File: SmartVitals_App/SmartVitals_App/src/usf/smartvitals/json/Models/Sample.java
package usf.smartvitals.json.Models;

import com.google.gson.annotations.SerializedName;

public class Sample {
	
	@SerializedName("SampleID")
	public int SampleID;
	
	@SerializedName("TestId")
    public int TestID;
	
	@SerializedName("Time")
    public String Time;
	
	@SerializedName("HeartRate")
    public double HeartRate;
	
	@SerializedName("EKG")
    public double EKG;
	
	@SerializedName("BloodPressure")
    public double BloodPressure;
	
	@SerializedName("SkinTemperature")
    public double SkinTemperature;
	
	@SerializedName("RespirationRate")
    public double RespirationRate;
	
	@SerializedName("Posture")
    public double Posture;
	
	@SerializedName("AccX")
    public double AccX;
	
	@SerializedName("AccY")
    public double AccY;
	
	@SerializedName("AccZ")
    public double AccZ;
}

File: SmartVitals_Server/SmartVitals_Server/Models/Sample.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using System.ComponentModel.DataAnnotations;
using SmartVitals.Services.Models;

namespace SmartVitals_Server.Models
{
    public class Sample
    {
        public int SampleId { get; set; }
        public int TestId { get; set; }
        public String Time { get; set; }
        public double HeartRate { get; set; }
        public double RespirationRate { get; set; }
        public double Posture { get; set; }
        public double EKG { get; set; }
        public double BloodPressure { get; set; }
        public double SkinTemperature { get; set; }
        public double AccX { get; set; }
        public double AccY { get; set; }
        public double AccZ { get; set; }
    }
}
